test/js_builds/shared_tests.dart

```diff
@@ -0,0 +1,147 @@
+// ignore_for_file: deprecated_member_use_from_same_package
```

Comment:
This file looks like it's all new and the other file was deleted, but it was just renamed with content added.

I strongly recommend reviewing [all commits up to the rename commit](https://github.com/cleandart/react-dart/pull/293/files/e2fe5a4465bbd738c6a3e4e2e5241f4a90f2280a) separately from [the rename commit itself](https://github.com/cleandart/react-dart/pull/293/commits/1692efb1d089654bc9159435ae5aa387a4ad6a27) for a better diff.